packages/@uppy/dashboard/src/components/AddFiles.js

```diff
@@ -125,21 +149,36 @@ class AddFiles extends Component {
     )
   }
 
+  renderAcquirers = (acquirers) => {
+    // Group last two buttons in <nobr>, so we don’t end up with
+    // just one button on a new line
+    const acquirersWithoutLastTwo = [...acquirers]
+    const lastTwoAcquirers = acquirersWithoutLastTwo.splice(acquirers.length - 2, acquirers.length)
+
+    return (
+      <div class="uppy-DashboardTabs-list" role="tablist">
+        {this.renderMyDeviceAcquirer()}
+        {acquirersWithoutLastTwo.map((acquirer) => this.renderAcquirer(acquirer))}
+        <nobr role="presentation">
```

Comment:
MDN is not a fan of nobr: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nobr
should we use a span with `white-space: nowrap` instead?